docs/components/plugins/api/APISectionUtils.tsx

```diff
@@ -67,8 +67,6 @@ const nonLinkableTypes = [
   'FileList',
   'Manifest',
   'NativeSyntheticEvent',
-  'Omit',
-  'Pick',
```

Comment:
Is `'Partial'` missing here?

Reply:
I wasn't on the non-linkable list, because it's the first use case of `Partial` in the converted packages (~45 packages already converted).

Reply:
Okay, thanks for explaining